config/initializers/app_info.rb

```diff
@@ -3,4 +3,57 @@
 module AppInfo
   GIT_REVISION = ENV.fetch('GIT_REVISION', 'MISSING_GIT_REVISION')
   GITHUB_URL   = 'https://github.com/department-of-veterans-affairs/vets-api'
+
+  def self.postgres_up
```

Comment:
Oh you know what.. I didn't realize that app_info was an initializer! We may want to move this to a separate module. It just naturally doesn't make sense for this method to live here

Reply:
@LindseySaari a new module within `config/initializers`?

Reply:
I'd say, maybe store it in lib. app/controller/concerns would be another choice, but since this is so simple and not a true concern file, just storing it in lib is probably the best option. 

module MyUtilityModule
  def self.postgres_up
     ## Code here
   end
 end
 
Since it'll be stored in lib, you'll have to do a require at the top of the controller file, then you can just call `AppInfo.report_info`
 
 Initializers are used

Reply:
Fixed. Thanks for the feedback!